### PR TITLE
Blog: ParallelAccelerator guest post.

### PR DESCRIPTION
**Pull request by lkuper**
@ViralBShah Here's our guest post with a few edits to the last version
you saw.  For the images, I've tried to use URLs that will eventually
be correct, but let me know if I should be doing something else.

Thanks!

### CODE REVIEW

**Comment by ViralBShah** [comment]
Thanks! We'll wait for a day or so for any comments and then merge.

**Comment by lkuper** [comment]
@johnmyleswhite Thanks!

BTW, is there any way we can have https://github.com/JuliaLang/METADATA.jl/pull/4692 merged before this goes up?  I want the package to be as up-to-date as possible before we attract any attention to it. :)

**Comment by johnmyleswhite** [comment]
Done. Hopefully @tkelman will yell at me if the passing tests weren't sufficient to detect any problems.

**Comment by ViralBShah** [comment]
I say its good to merge too now. @lkuper Would be great if you can squash the commits, and give the green signal from your side.

**Comment by lkuper** [comment]
OK, I think everything's correctly squashed now. Go for it.

File: blog/_posts/2016-03-01-parallelaccelerator.md [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+### Stencil
+
+In addition to map, reduce, and comprehension, ParallelAccelerator
+targets a fourth parallel pattern:
+[stencil computations](https://en.wikipedia.org/wiki/Stencil_code).  A
+stencil computation updates the elements of an array according to a
+fixed pattern called a stencil.  In fact, the `avg` comprehension
+example above could also be thought of as a stencil computation,
+because it updates the contents of an array based on each element's
+neighbors.  However, stencil computations differ from the other other
```
Comment (eford) [inline blog/_posts/2016-03-01-parallelaccelerator.md]: Delete one "other"

**Comment by lkuper** [comment]
@eford Thanks, good catch. Fixed and re-squashed.

**Comment by johnmyleswhite** [comment]
I'm going to merge this.

**Comment by lkuper** [comment]
Great, thanks!

**Comment by lkuper** [comment]
@johnmyleswhite Hmm. The code blocks seem to be rending incorrectly, and images are broken. It looks fine on https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/2016-03-01-parallelaccelerator.md.  I'll see if I can fix it.

**Pull request merged**